l10n_ro_partner_unique/models/res_partner.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# ©  2015 Forest and Biomass Services Romania
+# See README.rst file on addons root folder for license details
+
+from openerp import models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    sql_constraints = [
+        ('vat_nrc_uniq',
+         'unique (id)',
```

Comment:
@lepistone Yes, the constraint is in odoo base romanian localization, so i want to extend that one with the nrc field...